src/components/SRNEntry.vue

```diff
@@ -235,7 +236,8 @@ export default {
         }
       }
     },
-    /** This method is called whenever - button is clicked, to remove an input field
+    /** This method is called whenever - button is clicked.
```

Comment:
```suggestion
    /** This method is called whenever "-" button is clicked.
```

Reply:
Also it is super helpful if a reviewer makes a suggestion, one should look for places in the entire code where this suggestion could be applied.
Because I might've seen that this `minus` is not in quotes in one place, but I might not have checked other places. That is the job of the person who created the PR